modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/settings/data-privacy/hocs/RequestList.tsx

```diff
@@ -315,6 +319,9 @@ const RequestList: React.FC<IRequestListProps> = ({
 		});
 	};
 
+	const isCurrentUserAsiteMember =
+		currentUser.roleName === UserRoleNames.Member;
```

Comment:
Hey @fmarcio you can do it instead:

```suggestion
	const authorized = currentUser.isAdmin();
```

Reply:
@interaminense